bats/fb-content-katello.bats

```diff
@@ -109,7 +109,8 @@ setup() {
 }
 
 @test "install subscription manager" {
-  cat > /etc/yum.repos.d/candlepin.repo << EOF
+  if tIsRHEL 6; then
+    cat > /etc/yum.repos.d/subscription-manager.repo << EOF
```

Comment:
we can just drop the rhel6 support now, katello and foreman no longer support it.